docker.go

```diff
@@ -51,10 +55,77 @@ func dockerBuild(build schema.Build) error {
 	return nil
 }
 
+func dockerCopy(build schema.Build) (string, error) {
```

Comment:
I think `extractCache()` is better than `dockerCopy()`
`docker copy` is just an internal process in extracting the cache.